README.md

```diff
@@ -3,6 +3,7 @@
 > Gets an auth token for a repo via a GitHub app installation
 
 [![CircleCI](https://circleci.com/gh/electron/github-app-auth.svg?style=svg)](https://circleci.com/gh/electron/github-app-auth)
```

Comment:
```suggestion
[![CircleCI](https://circleci.com/gh/electron/github-app-auth.svg?style=shield)](https://circleci.com/gh/electron/github-app-auth)
```

make them match :D

Reply:
Yea, I'm more of a fa of the shield style for the CircleCI badges anyway. 👍